File: exercícios/lista9/ex2.scala
object ConversorTemperatura {
  
  def celsiusParaFahrenheit(celsius: Double): Double = {
    (celsius * 9 / 5) + 32
  }

  def main(args: Array[String]): Unit = {
    val celsius = 30.0
    val fahrenheit = celsiusParaFahrenheit(celsius)
    
    println(f"$celsius%.1f°C equivale a $fahrenheit%.1f°F")
  }
}

File: exercícios/lista11/ex2.hs
celsiusParaFahrenheit :: Float -> Float
celsiusParaFahrenheit c = (c * 9 / 5) + 32

main :: IO ()
main = do
  let celsius = 30.0
  let fahrenheit = celsiusParaFahrenheit celsius
  putStrLn ("Temperatura em Celsius: " ++ show celsius ++ "°C")
  putStrLn ("Temperatura em Fahrenheit: " ++ show fahrenheit ++ "°F")
